src/AppEntry.js

```diff
@@ -57,14 +56,17 @@ const AppEntry = ({ useLogger, connectToDevTools }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [client]);
 
-    return <IntlProvider locale={navigator.language.slice(0, 2)} messages={messages} onError={console.log}>
+    return <IntlProvider locale={navigator.language.slice(0, 2)} messages={messages} onError={() => undefined}>
         <ApolloProvider client={client}>
-            <Provider store={(useLogger ? init(logger) : init()).getStore()}>
-                <Router basename={getBaseName(window.location.pathname)}>
-                    <NotificationsPortal />
-                    <App />
-                </Router>
-            </Provider>
+            <RegistryContext.Provider value={{               getRegistry: () => registry
+            }}>
```

Comment:
We should style this a bit better
```suggestion
            <RegistryContext.Provider value={{ getRegistry: () => registry }}>
```